src/controller/CHIPDeviceController.cpp

```diff
@@ -1014,5 +1041,61 @@ void DeviceCommissioner::OnSessionEstablishmentTimeoutCallback(System::Layer * a
     reinterpret_cast<DeviceCommissioner *>(aAppState)->OnSessionEstablishmentTimeout();
 }
 
+CHIP_ERROR DeviceCommissioner::DiscoverAllCommissioning()
+{
+#if CHIP_DEVICE_CONFIG_ENABLE_MDNS
```

Comment:
Should these functions clear `mCommissionableNodes` array to remove stale results?